aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/events/SyncQueriesStartedEvent.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore.syncengine.events;
+
+/**
+ * Event payload emitted when the sync process starts.
+ */
+public final class SyncQueriesStartedEvent {
+    private String[] models;
+
+    /**
+     * Constructs a SyncQueriesStartedEvent object.
+     * @param models An array of model names.
+     */
+    public SyncQueriesStartedEvent(String[] models) {
+        this.models = models;
```

Comment:
If you use the array primitive, be sure to do an `Array.copy` on the input, so that your DTO won't be effected if the input hcanges. https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#copyOf(U[],%20int,%20java.lang.Class)

same deal on this one -- `equals`, `toString`, and `hashCode`, please

Reply:
Done. Added tests using `EqualsToStringHashValidator`